src/helper/redisWrapper.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace TourCMS\OnBoarding\Helper;
+
+use TourCMS\Core\RedisService as Redis;
+use TourCMS\OnBoarding\Config\env;
+
+class redisWrapper extends Redis
+{
+    public $expirationTime;
+
+
+    public function __construct()
+    {
+        parent::__construct(env::getEnvVariable("REDIS_HOST"), env::getEnvVariable('REDIS_PORT'), env::getEnvVariable('REDIS_PASSWORD'));
+        $this->expirationTime = strtotime(date("Y-m-d H:i:s", strtotime("+1 day")));
```

Comment:
is this used?

Reply:
Anyway, a good option, should be a function where you could pass the timestamp of expiration instead of hard-coding it in the code to 1 day